source/SAMRAI/tbox/MathUtilities.C

```diff
@@ -16,6 +16,13 @@
 namespace SAMRAI {
 namespace tbox {
 
+// initialize static private members?
+template <class TYPE>
+TYPE MathUtilities<TYPE>::s_zero = 0;
```

Comment:
This is breaking the build - ending up with multiple defines.

Reply:
Sorry about that. I could not reproduce the issue when building with `CMake`, but, I can certainly see how a host-code would run in to this. 

After closer inspection of this, it seems that the private ``s_zero`` and ``s_one`` members of the ``MathUtilities`` singleton are not really necessary. 

Please take a look at the changes in 467542c and let me know what you think.

Reply:
This looks good, thanks! I'll wait for @nselliott to review too.

Reply:
Sounds good. Thanks.